Add notebook with TF2 Object Detection model conversion OOB with MO Python API (#1154)
openvinotoolkit/openvino_notebooks: notebooks/120-tf2-object-detection-to-openvino/README.md

> # TensorFlow 2 Object Detection to OpenVINO™ Model Conversion Tutorial

Reviewer: ```suggestion
# Convert TensorFlow 2 Object Detection Models to OpenVINO™ 
```

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F120-tf2-object-detection-to-openvino%2F120-tf2-object-detection-to-openvino.ipynb)

<img src="https://github.com/openvinotoolkit/openvino_notebooks/assets/41733560/f9b59be1-1d2f-4e13-9678-67205be78841" width=300>

This tutorial explains how to convert [TensorFlow 2 Object Detection](https://github.com/tensorflow/models/tree/master/research/object_detection) models to OpenVINO IR.

## Notebook Contents

> The notebook shows how to convert the [Faster R-CNN with Resnet-50 V1 Object Detection model](https://tfhub.dev/tensorflow/faster_rcnn/resnet50_v1_640x640/1) and then detect objects on an image with OpenVINO Runtime.

Reviewer: ```suggestion
The notebook shows how to convert the [Faster R-CNN with Resnet-50 V1 Object Detection model](https://tfhub.dev/tensorflow/faster_rcnn/resnet50_v1_640x640/1) and then detect objects in an image using OpenVINO Runtime.
```

## Installation Instructions

> If you have not installed all required dependencies, follow the [Installation Guide](../../README.md).

Reviewer 2: ```suggestion
This is a self-contained example that relies solely on its own code.</br>
We recommend  running the notebook in a virtual environment. You only need a Jupyter server to start.
For details, please refer to [Installation Guide](../../README.md).
```

Author: Wording updated
